src/snowflake/cli/_plugins/nativeapp/utils.py

```diff
@@ -96,3 +96,14 @@ def verify_no_directories(paths_to_sync: Iterable[Path]):
 def verify_exists(path: Path):
     if not path.exists():
         raise ClickException(f"The following path does not exist: {path}")
+
+
+def sanitize_dir_name(dir_name: str) -> str:
```

Comment:
Could this result in name conflicts in that case?

Reply:
Yes. In our case `os.makedirs()` will fail if there is a conflict as the directory will already exist. Added a more meaningful message for this case.